lexicons/app/bsky/feed/detach.json

```diff
@@ -0,0 +1,32 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.detach",
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
+      "description": "Record defining post URIs detached from a root post. The record key (rkey) of the detach record must match the record key of the root post in question, and that record must be in the same repository.",
+      "record": {
+        "type": "object",
+        "required": ["post", "targets", "updatedAt"],
+        "properties": {
+          "post": {
+            "type": "string",
+            "format": "at-uri",
+            "description": "Reference (AT-URI) to the post record."
+          },
+          "targets": {
+            "type": "array",
+            "maxLength": 50,
```

Comment:
you said "1k" in the PR description, but only 50 here; was that intentional? either is probably ok/plausible to me, if I was painting the bike shed i'd probably go with 200